src/main/java/at/hannibal2/skyhanni/config/features/Garden.java

```diff
@@ -1024,10 +1024,46 @@ public static class CustomSpeed {
     public Position farmingFortuneForContestPos = new Position(180, 156, false, true);
 
     @Expose
-    @ConfigOption(name = "Yaw / Pitch", desc = "Displays yaw and pitch with 4-digit accuracy while holding a farming tool. Automatically fades out if there is no movement for 3 seconds.")
+    @ConfigOption(name = "Yaw / Pitch", desc = "")
+    @ConfigEditorAccordion(id = 23)
+    public boolean yawpitch = false;
+
+    @Expose
+    @ConfigOption(name = "Enable", desc = "Displays yaw and pitch with 4-digit accuracy while holding a farming tool. Automatically fades out if there is no movement for 3 seconds.")
```

Comment:
With the new settings, the description could be shortened to:
"Displays yaw and pitch while holding a farming tool. Automatically fades out if there is no movement."

Reply:
Right, didn't even read check the old one and just adjusted the name. Thanks for the hint!